src/recipes/tests/mocks/get-recipe-response.mock.ts

```diff
@@ -0,0 +1,34 @@
+import { GetRecipeResponse } from 'src/recipes/dto/get-recipe-response';
```

Comment:
this import should probably be fixed (so that it's relative - not starting from `src` folder)